.github/workflows/testing_multiversion.yml

```diff
@@ -1,17 +1,15 @@
-name: unit tests [Python 3.8|3.9|3.10]
+name: unit tests [Python 3.8|3.9|3.10|3.11]
 
 on:
   push:
-    branches: [ main ]
```

Comment:
If we are running this on every branch push, probably running the coveralls can also be done here.

Reply:
I think I just removed this for testing purposes, will put it back, thanks